app/experimenter/experiments/tests/test_views.py

```diff
@@ -286,6 +289,23 @@ def test_filters_experiments_with_surveys(self):
 
         self.assertEqual(set(filter.qs), set([exp_1, exp_2]))
 
+    def test_filters_for_subscribed_experiments(self):
+        exp_1 = ExperimentFactory.create(name="Experiment", slug="experiment")
+        ExperimentFactory.create()
+        ExperimentFactory.create()
+
+        exp_1.subscribers.add(self.user)
+
+        self.assertTrue(self.user in exp_1.subscribers.all())
+
+        filter = ExperimentFilterset(
+            {"subscribed": "on"},
+            request=self.request,
+            queryset=Experiment.objects.all(),
+        )
+
+        self.assertEqual(set(filter.qs), set([exp_1]))
```

Comment:
I don't think you need the set here, since there is only one item. You still need to ask the ORM to actually run the query though, and converting it to a list with `list()` will do that:

```suggestion
        self.assertEqual(list(filter.qs), [exp_1])
```

Reply:
Makes sense. I switched it to list like you suggested. I could also call .get() on it and just compare it to exp_1... can't tell if there's something inherently worse in doing that.

Reply:
I thought about `.get()` as well, and I didn't suggest it because it introduced a weird asymetry between this case and the case where multiples are returned. Either would work though.